jmbitcoin/jmbitcoin/secp256k1_transaction.py

```diff
@@ -106,70 +106,87 @@ def human_readable_output(txoutput):
         pass # non standard script
     return outdict
 
-def estimate_tx_size(ins, outs, txtype='p2pkh', outtype=None):
+def there_is_one_segwit_input(input_types):
```

Comment:
Add Python type hints here and elsewhere in new / modified functions?

Reply:
Addressed in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1421/commits/6250d243df9469b0e20a01499b0e5d91b4206800